components/script/dom/node.rs

```diff
@@ -2017,6 +2017,16 @@ impl Node {
         parent.owner_doc().remove_script_and_layout_blocker();
     }
 
+    // https://dom.spec.whatwg.org/multipage/#string-replace-all
+    pub fn string_replace_all(string: DOMString, parent: &Node) {
```

Comment:
Should this be a method on `node`?

Reply:
I went for consistency with replace_all: https://github.com/servo/servo/blob/728133611656f37480b96103dcc1d025f7d12ba3/components/script/dom/node.rs#L1973-L1974

Reply:
Oh, wait, I thought it wasn't on Node, and then was suggesting that it should be. I didn't notice that it was :)